src/fmu/settings/_resources/changelog_manager.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING, Self
+
+from fmu.settings._resources.log_manager import LogManager
+from fmu.settings.models._enums import FileName
+from fmu.settings.models.change_info import ChangeInfo
+from fmu.settings.models.log import Log
+
+if TYPE_CHECKING:
+    # Avoid circular dependency for type hint in __init__ only
+    from fmu.settings._fmu_dir import (
+        FMUDirectoryBase,
+    )
+
+
+class ChangelogManager(LogManager):
```

Comment:
```suggestion
class ChangelogManager(LogManager[ChangeInfo]):
```

The `LogManager` handles a `ChangeInfo`